libs/dal/src/repositories/organization/organization.repository.ts

```diff
@@ -58,6 +59,19 @@ export class OrganizationRepository extends BaseRepository<OrganizationDBModel,
     );
   }
 
+  async updateServiceLevel(organizationId: string, apiServiceLevel: ApiServiceLevelEnum) {
+    return this.update(
+      {
+        _id: organizationId,
+      },
+      {
+        $set: {
+          apiServiceLevel,
+        },
+      }
+    );
+  }
+
```

Comment:
This will be used to set a custom API service level when customers upgrade to higher product tiers. It's only used for testing now.